debian/rules

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/make -f
+
+include /usr/share/dpkg/default.mk
+
+# see FEATURE AREAS in dpkg-buildflags(1)
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+COMMON_CONFIGURE_OPTIONS =\
+  -DWLCS_BUILD_ASAN=ON \
+  -DWLCS_BUILD_UBSAN=ON
+
+$(info Building for $(DEB_DISTRIBUTION) on $(DEB_HOST_ARCH))
+
+ifeq ($(filter i386 armhf, $(DEB_HOST_ARCH)),)
+  ifeq ($(DEB_DISTRIBUTION),xenial)
+    ifneq ($(filter arm64 ppc64el, $(DEB_HOST_ARCH)),)
+      $(info Hello from xenial-arm64 or xenial-ppc64el)
+      # arm64 on 16.04 does not have tsan
+      COMMON_CONFIGURE_OPTIONS += -DWLCS_BUILD_TSAN=OFF
+    else
+      COMMON_CONFIGURE_OPTIONS += -DWLCS_BUILD_TSAN=ON
+    endif
+  else
+    COMMON_CONFIGURE_OPTIONS += -DWLCS_BUILD_TSAN=ON
+  endif
+else
+  # i386 and armhf do not have tsan
+  COMMON_CONFIGURE_OPTIONS += -DWLCS_BUILD_TSAN=OFF
+endif
```

Comment:
Couldn't we just add one of those to `COMMON_CONFIGURE_OPTIONS` above and only append the opposite if the conditions are met? It should be fine to override a definition later on the arguments list?

Reply:
Yup, that (appears to) works.